scripts/change_url

```diff
@@ -75,14 +75,17 @@ then
 	ynh_add_nginx_config
 fi
 
-# Change the domain for nginx
+# Change the domain for nginx and update Fail2ban
 if [ $change_domain -eq 1 ]
 then
 	# Delete file checksum for the old conf file location
 	ynh_delete_file_checksum "$nginx_conf_path"
 	mv $nginx_conf_path /etc/nginx/conf.d/$new_domain.d/$app.conf
 	# Store file checksum for the new config file location
 	ynh_store_file_checksum "/etc/nginx/conf.d/$new_domain.d/$app.conf"
+
+	# Update Fail2ban configuration
+	ynh_add_fail2ban_config --logpath="/var/log/nginx/$new_domain-error.log" --failregex="^.*authentication failure. while reading response header from upstream, client: <HOST>,.*POST $path_url.*$" --max_retry=5
```

Comment:
I think we have to update the configuration when only path_url is modified. So perhaps we should update the conf outside this `if`?

Reply:
Done